packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/mouseUtils/handleMouseDownEvent.ts

```diff
@@ -53,65 +54,67 @@ export function handleMouseDownEvent(
                     selection?.setBaseAndExtent(state.firstTarget, 0, state.lastTarget, 0);
                     selectTable(editor, state);
                 }
-
                 return;
             }
         }
     }
-    if (which == LEFT_CLICK && !shiftKey) {
-        clearState(state, editor);
+    if (which == LEFT_CLICK) {
+        if (!shiftKey && !tripleClick) {
+            clearState(state, editor);
 
-        if (getTableAtCursor(editor, event.rawEvent.target)) {
-            const doc = editor.getDocument() || document;
+            if (getTableAtCursor(editor, event.rawEvent.target)) {
+                const doc = editor.getDocument() || document;
 
-            const mouseUpListener = getOnMouseUp(state);
-            const mouseMoveListener = onMouseMove(state, editor);
-            doc.addEventListener('mouseup', mouseUpListener, true /*setCapture*/);
-            doc.addEventListener('mousemove', mouseMoveListener, true /*setCapture*/);
+                const mouseUpListener = getOnMouseUp(state);
+                const mouseMoveListener = onMouseMove(state, editor);
+                doc.addEventListener('mouseup', mouseUpListener, true /*setCapture*/);
+                doc.addEventListener('mousemove', mouseMoveListener, true /*setCapture*/);
 
-            state.mouseMoveDisposer = () => {
-                doc.removeEventListener('mouseup', mouseUpListener, true /*setCapture*/);
-                doc.removeEventListener('mousemove', mouseMoveListener, true /*setCapture*/);
-            };
+                state.mouseMoveDisposer = () => {
+                    doc.removeEventListener('mouseup', mouseUpListener, true /*setCapture*/);
+                    doc.removeEventListener('mousemove', mouseMoveListener, true /*setCapture*/);
+                };
 
-            state.startedSelection = true;
+                state.startedSelection = true;
+            }
         }
-    }
 
-    if (which == LEFT_CLICK && shiftKey) {
-        editor.runAsync(editor => {
-            const sel = editor.getDocument().defaultView?.getSelection();
-            const first = getCellAtCursor(editor, sel?.anchorNode);
-            const last = getCellAtCursor(editor, sel?.focusNode);
-            const firstTable = getTableAtCursor(editor, first);
-            const targetTable = getTableAtCursor(editor, first);
```

Comment:
@BryanValverdeU I saw L86 and L87 are actually the same. Please review if it is ok to remove this line. Thanks.

Reply:
I see, should be okay to remove it.